### PR TITLE
bug: Use RFC8282 Vapid by default

### LINKED ISSUE #74: support RFC 8292
The python code seems to supports [draft-ietf-webpush-vapid-01](https://tools.ietf.org/html/draft-ietf-webpush-vapid-01) and [draft-ietf-webpush-vapid-02](https://tools.ietf.org/html/draft-ietf-webpush-vapid-02), with draft 1 still the default.

However according to those links there are draft 3 and 4 and then [RFC8292](https://tools.ietf.org/html/rfc8292) (if I'm interpreting ieft.org correctly).

Are there any significant changes between draft 2 and RFC? If so could this library be updated to support the RFC?

### PR DESCRIPTION
**Pull request by jrconlin**
* Added pypi badge
* Updated pypi packaging

Closes #74

### CODE REVIEW

**Comment by jrconlin** [comment]
R? @samuelcolvin

File: python/py_vapid/main.py [hunk 1]
```diff
@@ -25,7 +25,7 @@ def main():
     parser.add_argument('--sign', '-s', help='claims file to sign')
     parser.add_argument('--gen', '-g', help='generate new key pairs',
                         default=False, action="store_true")
-    parser.add_argument('--version2', '-2', help="use VAPID spec Draft-02",
+    parser.add_argument('--version2', '-2', help="use RFC8292 VAPID spec",
```
Comment (samuelcolvin) [inline python/py_vapid/main.py]: maybe make this the default? default is `False` here, but `True` below.

Comment (jrconlin) [reply python/py_vapid/main.py]: ah, good catch, thanks!

**Comment by samuelcolvin** [comment]
lgtm

**Pull request merged**